docs/source/examples/3d.rst

```diff
@@ -19,3 +18,64 @@
             self.set_camera_orientation(phi=75 * DEGREES, theta=30 * DEGREES)
             self.add(axes, sphere)
 
+.. manim:: Example3DNo2
```

Comment:
It's hard to determine at first glance what is the difference between this one and the previous one.

1. A more descriptive class name would help.
2. A comment on L35 would help.